oci_test.go

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestChangeToBundlePath(t *testing.T) {
```

Comment:
After re-examining the code, I've decided to change the logic we have in `grpc.go`.
Instead of doing `changeToBundlePath` and then `writeSpecToFile`, I've swapped the order of these operations. The concept of bundle doesn't make sense if we don't have `config.json` serialized in a directory. And the spec struct in the Go code does not defined where the bundle should be. 

I think it makes more sense to first make `dirname(rootfs)` a valid bundle (by convention, could be any other directory), then `chdir` to this bundle.

Reply:
So the conlusion is that we don't need to treat absolute and relative paths differently. If `spec.Root.Path` is the recommended `rootfs`, then `.` will become an OCI bundle, and that's fine.

Reply:
notifying @eguzman3 in case he notices something wrong with my reasoning :)